_extras/guide.md

```diff
@@ -1,8 +1,40 @@
 ---
 layout: page
-title: "Instructors' Guide"
+title: "Instructor Notes"
 permalink: /guide/
```

Comment:
Should the permalink be changed? Does this require a template change somewhere as well?